oeps/oep-0014-proc-archive-repos.rst

```diff
@@ -91,6 +91,13 @@ these steps to see if anyone would like to take ownership of it:
 Archive Steps
 -------------
 
+These steps should be followed for all repos within the edX organization(forks
+included). After some experiments with keeping archived repos in the ``edx``
+organization, we've learned that having abandonned code show up in searches
+hinders work to undrestand the current state of the system and the risk around
+new work. In particualr deprecations or API changes. Thus the choice to move all
```

Comment:
new work. In particualr deprecations or API changes -> new work, particularly deprecations and API changes.